packages/compartment-mapper/src/import-hook.js

```diff
@@ -686,20 +692,13 @@ export function makeImportNowHookMaker(
 
     /** @type {ImportNowHook} */
     const importNowHook = moduleSpecifier => {
-      // many dynamically-required specifiers will be absolute paths owing to use of `require.resolve()` and `path.resolve()`
-      if (isAbsolute(moduleSpecifier)) {
-        const record = findRedirect({
-          compartmentDescriptor,
-          compartmentDescriptors,
-          compartments,
-          absoluteModuleSpecifier: moduleSpecifier,
-          packageLocation,
-        });
-        if (record) {
-          return record;
-        }
-        // if there is no record found this way, we will search for it instead of considering it to be an exit module
-      } else if (moduleSpecifier !== '.' && !moduleSpecifier.startsWith('./')) {
+      /**
+       * Attempt to load the moduleSpecifier via an {@link ExitModuleImportNowHook}, if one exists.
+       *
+       * If it doesn't exist, then throw an exception.
+       * @returns {VirtualModuleSource}
+       */
+      const tryExitModuleImportNowHook = () => {
```

Comment:
(could) I'm pretty sure there's nothing stopping us from not nesting this one within importNowHook

Reply:
(nit, not sure) I'm thinking the name could more explicitly remind us that the "cannot find" error is here
```suggestion
      const importExitModuleOrError = () => {
```